app/tray/wintray/messages.go

```diff
@@ -3,10 +3,12 @@
 package wintray
 
 const (
-	firstTimeTitle   = "Ollama is running"
-	firstTimeMessage = "Click here to get started"
-	updateTitle      = "Update available"
-	updateMessage    = "Ollama version %s is ready to install"
+	firstTimeTitle    = "Ollama is running"
+	firstTimeMessage  = "Click here to get started"
```

Comment:
I think this may need a rebase

Reply:
Just rebased and didn't pick up any changes here.  What were you expecting?